### DISCUSSION TITLE
What temp advance unlocking do?

### DISCUSSION THREAD

**Discussion by pokegamer5547** [Kunzisoft/KeePassDX#810] [Q&A]
It's saying it will delete some kind of encryption keys which makes me worry about using it. What exactly it does?

**Comment by J-Jamet** [comment]
This indicates that it will delete the currently saved keys that allow the use of biometric recognition or device credential.
Concretely, it simply means that you will have to re-register the biometric recognition or the device credential with your main password because the configuration of the app internally changes.

**Reply by pokegamer5547** [reply]
Oh I see, thankyou so much for explaining to me :) 
Can I add biometric again after doing this?

**Reply by J-Jamet** [reply]
Yes of course, in fact the message is only a warning. Internally, the memory switches to temporary memory in a service. "temp" is "temporary" but the fingerprint operation remains the same, it's just that after a while the encrypted keys are deleted and it is necessary to re-link the fingerprint to the password. It may not be clear at this level.
